### ISSUE TITLE
[3.2.2] Debian package is broken (wrong path).

### ISSUE THREAD

**Issue by tmartincpp** [RocketChat/Rocket.Chat.Electron#2044]
## My Setup

- Operating System: Debian 10 (Buster)
- App Version: 3.2.2
- Installation type: .deb from releases page

- [x] I have tested with the latest version
- [x] I can simulate the issue easily

## Description
Debian package is broken with 3.2.2.

It seems that default Rocket.Chat.Electron's directory has changed (from /opt/rocketchat-desktop/ to /opt/Rocket.Chat/) but this change has not been applied to the Debian package (neither symlink or postinst).

Examples :
```
$ ll /usr/bin/rocketchat-desktop
lrwxrwxrwx 1 root root 42 May 17 18:40 /usr/bin/rocketchat-desktop -> /opt/rocketchat-desktop/rocketchat-desktop
$ ll /opt/rocketchat-desktop/rocketchat-desktop
ls: cannot access '/opt/rocketchat-desktop/rocketchat-desktop': No such file or directory
```
```
$ /opt/Rocket.Chat/rocketchat-desktop 
[20981:0517/184150.034255:FATAL:setuid_sandbox_host.cc(158)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /opt/Rocket.Chat/chrome-sandbox is owned by root and has mode 4755.
Trace/breakpoint trap
```

### Current Behavior
RC desktop cannot be started without manual operations.

### Expected Behavior
RC desktop should work out of box.

**Comment by tmartincpp** [comment]
@tassoevan : this bug is still present in 3.2.3.

**Comment by tmartincpp** [comment]
I think I found a workaround : removing _executableName_ from the _electron-builder.json_ works but it will produce a new installation path: _/opt/Rocket.Chat/rocketchat_ which should work.
```
--- ../old/Rocket.Chat.Electron-3.1.1.orig/electron-builder.json	2020-12-25 16:12:20.000000000 +0100
+++ electron-builder.json	2021-07-05 13:38:02.273129103 +0200
@@ -114,7 +114,6 @@
       "rpm",
       "snap"
     ],
-    "executableName": "rocketchat-desktop",
     "category": "GNOME;GTK;Network;InstantMessaging",
     "desktop": {
       "StartupWMClass": "Rocket.Chat",
```

I'm unable to find a better fix.

**Comment by tmartincpp** [comment]
Fixed in 3.4.0.
